examples/correlation_matrices/test.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/bash
```

Comment:
you can add these commands in a README instead, we do not have such scripts in the library

Reply:
This is fixed.